Bot.Core/Announcer.cs

```diff
@@ -19,10 +19,10 @@ public class Announcer : IAnnouncer
         IVersionProvider m_versionProvider;
         IAnnouncementDatabase m_announcementDatabase;
         ITownDatabase m_townDatabase;
-        ICallbackScheduler<Queue<TownKey>> m_callbackScheduler;
         IBotSystem m_botSystem;
         IBotClient m_botClient;
         IDateTime m_dateTime;
+        ITownMaintenance m_startupTownTasks;
```

Comment:
I'd change the name of this service (looks like you changed the interface name but didn't update the member).